releasenotes/notes/add-openshift-profile.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+releaseNotes:|
+  **Added** an experimental OpenShift Kubernetes platform profile to `istioctl`. To install with the OpenShift profile, use
+  `istioctl install --set profile=openshift`.
+docs:
+- docs/setup/platform-setup/openshift/
```

Comment:
```
- |
  [openshift] docs/setup/platform-setup/openshift/
- |
  [profiles] docs/setup/install/istioctl/#install-a-different-profile
```

Would produce:

- **Added** an experimental OpenShift Kubernetes platform profile to `istioctl`. To install with the OpenShift profile, use
`istioctl install --set profile=openshift`.
 ([openshift]( docs/setup/platform-setup/openshift/
))([profiles]( docs/setup/install/istioctl/#install-a-different-profile
))

Reply:
Note that because the links are not absolute, they would be broken in any preview. I'd been assuming that the links would be absolute, but perhaps that's an invalid assumption.

Reply:
@brian-avery the tempalte could use improved instructions around docs. I am not quite clear, even from your comment, what to change, although I am a bit tired.

Just make it absolute? - i.e. lead off with a `/`?

Cheers,
-steve